docs/custom_virtualenvs.md

```diff
@@ -77,6 +77,48 @@ index d69e2c9..a08bae5 100644
 +    VENV_BASE=/var/lib/awx/venv make requirements_custom && \
 ```
 
+Kubernetes Custom Virtualenvs
+=============================
+
+You can create custom virtualenvs without updating the awx images by using initContainers and a shared emptydir within Kubernetes.  To start create an emptydir volume in the volumes stanza.
+
+    volumes:
+        - emptyDir: {}
+          name: custom-venv
+
+Now create an initContainer stanza.  You can subsititute your own custom images for this example we are using centos:7 as the base to build upon.  The command stanza is where you will add the python modules you require in your virtualenv.
+
+    initContainers:
+        - image: 'centos:7'
+        - name: init-my-custom-venv
```

Comment:
The extra dashes here are going to cause problems, I think it should look like:

```
initContainers:
   - image: ...
     name: ....
     command:
        - sh
        - ....
```

Reply:
Correct.  Updated.